gnovm/pkg/gnolang/stacktrace.go

```diff
@@ -0,0 +1,81 @@
+package gnolang
+
+import (
+	"fmt"
+	"strings"
+)
+
+const maxStacktraceSize = 128
+
+type Execution struct {
+	Stmt  Stmt
+	Frame *Frame
+}
+type Stacktrace struct {
+	Executions      []Execution
+	NumFramesElided int
+}
```

Comment:
I don't like Execution, as a name.

What about `StackTrace` / `CallTrace`?

Reply:
We already used Stacktrace for the parent struct. What do you think about this ?

```
type Trace struct {
	Stmt  Stmt
	Frame *Frame
}
type StackTrace struct {
	Traces      []Trace
	NumFramesElided int
}
```

Reply:
Yeah, I was suggesting for the two of them.

Actually what do you think about using `StackTrace` but changing `Execution` to `StackTraceCall`, and then have `Calls []StackTraceCall`?

Each of the item is a function call, so I thnk it makes sense.

I think the more appropriate term here would be "frame", but I don't want it to be confused with the Gno Frame. Maybe we could have `StackTraceFrame`

Reply:
> Actually what do you think about using StackTrace but changing Execution to StackTraceCall, and then have Calls []StackTraceCall?

Agree for this one.
Done here: https://github.com/gnolang/gno/pull/2145/commits/0bbc0c2e4c69291bbd34865932cea3f0aac591a8